cli/command/image/prune.go

```diff
@@ -42,6 +43,7 @@ func NewPruneCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.force, "force", "f", false, "Do not prompt for confirmation")
 	flags.BoolVarP(&opts.all, "all", "a", false, "Remove all unused images, not just dangling ones")
+	flags.BoolVarP(&opts.keepDownloadCache, "keep-download-cache", "k", false, "Don't clean out the download cache")
```

Comment:
I wonder if it would be clearer to instead have an option for purging the download cache that defaults to true. I don't have strong feelings about this and don't know what our conventions on this are, but thought I'd mention it for discussion. Most likely it's fine the way it is.